lessons/git/intro/example_file.md

```diff
@@ -0,0 +1,5 @@
+# UofTCoders Git Intro Lesson on October 5, 2017
+*This is a new text file for demonstration purposes*
+
+* This is Item 1 that laready exists
```

Comment:
already

Reply:
Could I leave it as is and use it as an example of a changed file instead?

Reply:
Sure, I did not realize that you would be corrected this in class